Sasquatch/Sasquatch/cs.lproj/Sasquatch.strings

```diff
@@ -0,0 +1,8 @@
+/*
+  Sasquatch.strings
+  Sasquatch
+*/
+"distribute_alert_title" = "New update available!";
+"distribute_alert_message" = "This is a customized dialog. - Czech";
+"distribute_alert_yes" = "Yes";
+"distribute_alert_no" = "No";
```

Comment:
We're not localizing sasquatch strings, why are these files here?

Reply:
This is required because the app supports all localized languages in Distribute.